modules/social_features/social_activity/social_activity.install

```diff
@@ -260,3 +261,56 @@ function social_activity_update_8807() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Add a new filter to "notifications" view.
+ */
+function social_activity_update_8901() {
```

Comment:
Would be great if we can update this to 10 number range as we are probably not backporting this feature.